apps/src/blockly/blocks/procedures_defnoreturn.js

```diff
@@ -0,0 +1,26 @@
+// Used to customize function definition blocks for Google Blockly labs.
+
+export default function() {
+  const initName = Blockly.Procedures.findLegalName('', this);
+  const nameField = new Blockly.FieldTextInput(
+    initName,
+    Blockly.Procedures.rename
+  );
+  nameField.setSpellcheck(false);
+  this.appendDummyInput()
+    .appendField(nameField, 'NAME')
+    .appendField('', 'PARAMS');
+  if (
```

Comment:
What are all of these different permutations?

Reply:
Oh, great catch! We actually don't need these. They're to allow for procedures to have comments like what is shown below. This isn't a feature we've ever used which is why comments are actually disabled elsewhere.
<img width="235" alt="image" src="https://user-images.githubusercontent.com/43474485/195867386-6c48636c-e8c5-46a0-ae60-db45261d579b.png">